migrations/0001-init-all-tables-f47bad.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2016 Eugene Frolov <eugene@frolov.net.ru>
+# Copyright 2025 Genesis Corporation
+#
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from restalchemy.storage.sql import migrations
+
+
+class MigrationStep(migrations.AbstarctMigrationStep):
+
+    def __init__(self):
+        self._depends = [
+            "0000-init-iam-tables-9c9153.py",
```

Comment:
Do we really want to use tree-like dependencies? I thought we usually use linear dependencies, in this case there should be one 0000 migration.

Reply:
I want us to be able to conveniently debug logically separated code segments at this stage. Reloading the database of a specific logical module is useful during development. For example, if I’m working on IAM functionality, I can just run `ra-rollback -m iam && ra-apply -m iam`. The rest of the database remains untouched, and I won’t have to manually push data into artifacts or nodes. Additionally, splitting the project into separate repositories when needed would make things easier to manage.